src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,70 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("String 클래스에 대한 학습 테스트")
+public class StringTest {
+
+    @DisplayName("'1,2'를 ,로 split 했을 때 1과 2로 잘 분리되는지 확인")
+    @Test
+    void split_comma() {
+        // given
+        String string = "1,2";
+        // when
+        String[] strings = string.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?